examples/draft.json

```diff
@@ -0,0 +1 @@
+{"statementGroups":[{},{},{},{}]}
```

Comment:
Everything at top-level in /examples uses the old statementGroups model and can be removed.